docs/assistant/cloud/libraries/overview.md

```diff
@@ -0,0 +1,17 @@
+---
+id: overview
+title: Libraries Overview
+sidebar_label: Overview
+slug: /cloud/libraries/
+---
+
+You can add colors and images (.jpg, .png, .svg .giff) as well as templates shared by the community to your libraries from the Assistant Pro Cloud.
+
+You can add images, colors, theme settings, and page content (posts, pages, Gutenberg Blocks and Beaver Builder Saved Content) to libraries from the Assistant WordPress Plugin.
+
+Assistant also supports the templates and saved content of most popular page editors including Divi, Visual Composer and Elementor.
+
+
+* [Adding assets overview](adding-assets/overview.md)
```

Comment:
Consistency: KB uses the imperative, not gerund, in titles with procedures to distinguish them from conceptual articles.

Reply:
My grammar is atrocious so I'm at a loss here.  @nancyhild, will you please make a suggestion for updated copy?